libfreerdp/gdi/gfx.c

```diff
@@ -85,11 +85,25 @@ static UINT gdi_ResetGraphics(RdpgfxClientContext* context,
 	}
 
 	free(pSurfaceIds);
+	{
+		UINT32 flags = FREERDP_CODEC_ALL;
 
-	if (!freerdp_client_codecs_reset(gdi->context->codecs, FREERDP_CODEC_ALL,
-	                                 gdi->width, gdi->height))
-		goto fail;
+		if (settings->RemoteFxCodec == FALSE)
```

Comment:
What is this for? No real reason to do that magic

Reply:
you are right, this flags does not make much sense on reset as it check for null ptr of each codec context,  looks like i am i bit overdone here.

Reply:
ah, it's just typo, i missed  what flags var not used at all...

Reply:
sorry for delay, i was very busy during last week,  typo fixed now, changes rebased against latest master.

Reply:
See below in how to unify this